tools/airgap_linter.py

```diff
@@ -14,8 +14,13 @@
 
 
 def extract_uris(file_name):
-    with open(file_name, "r") as file:
-        lines = file.readlines()
+    try:
+        with open(file_name, "r", encoding='utf8',) as file:
+            lines = file.readlines()
+    except UnicodeDecodeError as e:
```

Comment:
We should probably do this validation in `get_files_to_check_for_uris` so that we keep things DRY, but this would require some more refactoring.

Reply:
It seems as if we could just add the following function:
```python
def readlines_if_text_file(filename: str) -> List[str]:
    try:
        with open(filename, "r", encoding="utf8",) as fp:
            return file.readlines()
    except UnicodeDecodeError as e:
        print("The error message from below")
        return [] 
```

we can then safely call this function in `exctract_uris` and in the `for file_name in files:` loop in `validate_images` as:
```python
lines = readlines_if_text_file(file_name)
```
as the code in both cases seems to deal with the resultant empty list case correctly.